eng/common/templates/job/job.yml

```diff
@@ -132,7 +132,7 @@ jobs:
       displayName: RichCodeNav Upload
       inputs:
         languages: 'csharp'
-        environment: ${{ coalesce(parameters.richCodeNavigationEnvironment, 'prod') }}
+        environment: ${{ coalesce(parameters.richCodeNavigationEnvironment, 'production') }}
```

Comment:
If there's a bug here, it should be fixed in the dotnet/arcade repo

Reply:
Agreed! I only made this change here to test temporarily. See [this PR](https://github.com/dotnet/arcade/pull/6345) for this and other improvements in the arcade template